### PR TITLE
Added the possibility to store template data in files #FE-4332

### PR DESCRIPTION
**Pull request by ClementBouvierN**
- In release note :
  -  In chapter :  Features Bugs Tasks
  -  Text : #4332 It is now possible to push json files in businessconfig/businessdata and to access those files from templates

### CODE REVIEW

**Comment by freddidierRTE** [comment]
Only admin shall access the Post Method , to add in services/cards-consultation/src/main/java/org/opfab/cards/consultation/configuration/webflux/WebSecurityConfiguration.java
Add a karate test to check a none admin user cannot push a file

File: ui/main/src/app/modules/core/application-loading/application-loading.component.ts [hunk 1]
```diff
@@ -250,7 +252,17 @@ export class ApplicationLoadingComponent implements OnInit {
...
+        this.activityAreaChoiceAfterLoginComponent
+            .isFinishedWithoutError()
+            .subscribe(() => this.finalizeApplicationLoading());
+    }
+
+    private initOpFabAPI(): void {
+        const that = this;
+        opfab.businessconfig.businessData.get = async function (resourceName) {
+            const resource = await that.businessDataService.getBusinessData(resourceName);
+            return resource;
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/core/application-loading/application-loading.component.ts:264]: Clean code : method shall be declare after  finalizeApplicationLoading
